torchvision/prototype/transforms/_augment.py

```diff
@@ -40,24 +40,22 @@ def __init__(
             raise ValueError("Scale should be between 0 and 1")
         self.scale = scale
         self.ratio = ratio
-        self.value = value
+        if isinstance(value, (int, float)):
+            self.value = [value]
+        elif isinstance(value, str):
+            self.value = None
+        elif isinstance(value, tuple):
+            self.value = list(value)
+        else:
+            self.value = value
```

Comment:
Moving `value` handling from run-time to constructor. This is more code-quality than perf.